content/options.html

```diff
@@ -22,7 +22,45 @@
 
   <label for="github-username">Github Username</label>
   <input type="text" id="github-username" data-type="ghuser">
+
+  <h1>Working Hours</h1>
+  <br>
+
+  <label for="working-hours-checkbox">Only show review count during working hours</label>
+  <input type="checkbox" id="working-hours-checkbox">
+
+  <fieldset id="working-hours" disabled>
+    <label for="start-time">Start time</label>
+    <input type="time" id="start-time" value="09:00">
+    <br>
+    <label for="end-time">End time</label>
+    <input type="time" id="end-time" value="17:00">
+
+    <div id="days">
+      <p>Working Days:</p>
+      <label for="sunday">Sunday</label>
```

Comment:
Let's put the checkboxes before the labels here too. If they can be lined up properly, so much the better.